openshift-java/Jenkinsfile

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env groovy
+
+node('rhel7'){
+	stage('Checkout repo') {
+		deleteDir()
+		git url: 'https://github.com/redhat-developer/vscode-openshift-extension-pack/'
+        dir: 'openshift-java'
+	}
+
+	stage('Install requirements') {
+		def nodeHome = tool 'nodejs-8.11.1'
+		env.PATH="${env.PATH}:${nodeHome}/bin"
+		sh "npm install -g typescript vsce"
+	}
+
+	stage('Build') {
+		sh "npm install"
+		sh "npm run build"
```

Comment:
There is no build script in package.json.

Reply:
Removed the build action.